backend/test_observer/data_access/models_enums.py

```diff
@@ -35,6 +35,6 @@ class TestExecutionStatus(Enum):
     NOT_TESTED = "NOT_TESTED"
 
 
-class ArtefactStatus(Enum):
+class ArtefactStatus(str, Enum):
```

Comment:
sorry if it's a stupid question, but why do you inherit `str` as well here but not in other enums?